Ryujinx.Common/Configuration/ConfigurationFileFormat.cs

```diff
@@ -14,7 +14,7 @@ public class ConfigurationFileFormat
         /// <summary>
         /// The current version of the file format
         /// </summary>
-        public const int CurrentVersion = 12;
+        public const int CurrentVersion = 13;
```

Comment:
```suggestion
        public const int CurrentVersion = 14;
```
Version 13 is reserved for LDN